cli/ml_training.go

```diff
@@ -37,7 +38,7 @@ func MLSubmitCustomTrainingJob(c *cli.Context) error {
 
 	trainingJobID, err := client.mlSubmitCustomTrainingJob(
 		c.String(datasetFlagDatasetID), c.String(mlTrainingFlagName), c.String(mlTrainingFlagVersion), c.String(generalFlagOrgID),
-		c.String(trainFlagModelName), c.String(trainFlagModelVersion))
+		c.String(trainFlagModelName), c.String(trainFlagModelVersion), c.String(trainCliArgs))
```

Comment:
I think this could be a string slice? That way we won't have to do the splitting based on commas later

Reply:
I don't think it would. Doesn't the API take in a map in either case ([here](https://github.com/viamrobotics/api/blob/f58873f177e4c4551b5ca1a4e72f8f933b45ee57/proto/viam/app/mltraining/v1/ml_training.proto#L72))? How we're inputting arguments to the CLI is independent of how we pass along the parameters to the implemented RPC. We can make it convenient for the CLI to parse a list of comma separated strings (what the string slice functionality does) without needing to change the underlying API implementation. This line is just a convenience for mapping the CLI inputs to the SubmitCustomTrainingJobRequest. The actually gRPC request is made [here](https://github.com/viamrobotics/rdk/blob/e0ee2ab3d03a5823f2807d4b7503c52e693e051d/cli/ml_training.go#L143)